viewer/src/main/java/nl/b3p/viewer/stripes/EditFeatureActionBean.java

```diff
@@ -294,12 +299,13 @@ public Resolution saveRelatedFeatures() throws JSONException {
                         json.put("success", Boolean.TRUE);
                     } catch (Exception ex) {
                         log.error(String.format("cannot save relatedFeature Exception: ",ex));
```

Comment:
missing format specifiers, just use `log.error("cannot save relatedFeature Exception: ",ex);`

Reply:
Fixed in b352f9c66